### ISSUE TITLE
Bulk Select Not Working - when upgrading Nova to v2.6

### ISSUE THREAD

**Issue by kombimedia** [classic-o/nova-media-library#18]
**Describe the bug**
Hi @classic-o , thanks for your amazing plugin! However, when I click 'bulk select' from within the media library I am only able to select one item. I also notice that the bulk select action buttons are not displayed.

**To Reproduce**
1. Go to 'Media Library'
2. Click on 'Bulk select'
3. Select image/file

**Screenshots**
![Screenshot 2019-10-31 at 4 41 55 PM](https://user-images.githubusercontent.com/28551935/67918456-438d2080-fc01-11e9-885f-2a9e897b7222.png)

**Additional context**
I see the same issue when I 'open library' from within a resource.

Laravel v. 6.4.0
Nova v. 2.6.1
Media Library v. 0.5.2

Thanks for your help

**Comment by classic-o** [comment]
Hi. Can you show console errors?
Becouse on my localhost all works. (last versions of Nova and Laravel)

**Comment by kombimedia** [comment]
Hi @classic-o 

Thanks for coming back so quickly. No console errors..

<img width="1439" alt="Screenshot 2019-10-31 at 9 34 58 PM" src="https://user-images.githubusercontent.com/28551935/67931328-71d12700-fc26-11e9-8a8b-040d1afab9a0.png">

**Comment by classic-o** [comment]
I dont know what happend :(
Try install empty Laravel + Nova + My plugin without extra plugins\modules
I think, some of your modules disable my Library....

May be you do something else? Becouse when I do your steps, all work on my site.

**Comment by ArroWsGM** [comment]
I can confirm this.
Its connected to

> Improved checkbox handling and accessibility.

in [Nova 2.6.0](https://nova.laravel.com/releases/2.6.0), now checkboxes emits `input` event instead of `change`

**Comment by classic-o** [comment]
I have Nova 2.6.1 and all works good

**Comment by kombimedia** [comment]
@classic-o  must be a conflict somewhere in my configuration. I'll run some tests and come back to you.
@ArroWsGM  are you running any of the packages in the attached image?

![Screenshot 2019-11-01 at 7 03 16 AM](https://user-images.githubusercontent.com/28551935/67973677-cfd92b00-fc75-11e9-90cf-586730844868.png)

Cheers,

**Comment by ArroWsGM** [comment]
@classic-o , you need to rebuild assets to get this bug.

Fix as simple as
```vue
// /resources/js/module/Index/parts/action.vue
<checkbox-with-label
                class="mr-6"
                v-if="$parent.tool || $parent.listing"
                :checked="$parent.bulk.is"
                @input="changeBulk" //<-changed from `@change`
        >{{ __("nml_bulk_select") }}
</checkbox-with-label>
```
BTW, simple checkbox component was changed too, and you also need to adjust styles for items

@kombimedia this bug is not related to any other packages except Nova

**Comment by kombimedia** [comment]
Hi @classic-o 

I can confirm that when I downgrade Nova to v.2.5 'Bulk select' works as expected. See attached.

![Screenshot 2019-11-01 at 7 17 19 AM](https://user-images.githubusercontent.com/28551935/67974776-d8cafc00-fc77-11e9-965f-5a2fc88a49e4.png)

This may still be an issue with another package conflicting so I'll continue digging and let you know.

Cheers

**Comment by kombimedia** [comment]
@ArroWsGM  ok cool, thanks for the fix.

**Comment by classic-o** [comment]
Fixed in 0.5.3
I leave both variants for all Nova versions (@input and @change)

**Comment by pxlrbt** [comment]
@classic-o The fix is on master branch but release 0.5.3 does not contain the fix.

**Comment by classic-o** [comment]
Sorry, my bad. Fixed in 0.5.4
